src/shared/get-request-logger.js

```diff
@@ -3,7 +3,14 @@ import type {$Request} from "express";
 import type {Logger, RequestWithLog} from "./types.js";
 
 export const getRequestLogger = <TReq: RequestWithLog<$Request>>(
-    defaultLogger: Logger,
+    defaultLogger: ?Logger,
     request?: TReq,
-): Logger =>
-    request != null && request.log != null ? request.log : defaultLogger;
+): Logger => {
+    if (defaultLogger == null) {
+        if (request == null || request.log == null) {
+            throw new Error("No logs available");
+        }
```

Comment:
Added this since there's now a possibility of there being no root logger.

This should never happen though as the circumstances where that's the case, we only call this with our own instance of a default logger.